example.tfvars

```diff
@@ -16,7 +16,7 @@ namespace_list = [
 ]
 
 ## OPTIONAL - A randomly generated alphabetical string used to autheticate between Lightstep and your AWS account. Default: <none>
```

Comment:
maybe update the comment - Default: <none> is now Default: "foo"? or were you actually meaning to keep it commented out?

Reply:
oops - left some forceps in the patient